src/index.js

```diff
@@ -0,0 +1,47 @@
+import './styles.css';
+
+const list = document.querySelector('#todo-data');
+
+const data = [
+  {
+    description: 'Go swimming',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Create an animated puppet',
```

Comment:
Please do not implement all the functionality of the list here in this file as it is the entry file for the webpack. You may create another file and import that file here as a module.